### PR TITLE
Default ntp servers

### PR DESCRIPTION
**Pull request by jreidinger**
trello: https://trello.com/c/HtBLfvR0/4499-ostumbleweed-p5-1180699-microos-defaults-to-using-the-suse-ntp-pool
bzilla: https://bugzilla.suse.com/show_bug.cgi?id=1180699

### CODE REVIEW

File: control/control.rng [hunk 1]
```diff
@@ -607,6 +608,16 @@ by default true</a:documentation>
       <ref name="BOOLEAN"/>
     </element>
   </define>
+  <define name="default_ntp_servers">
+    <a:documentation>Defines list of default ntp servers from which product can pick.
+Used mainly to distinguish between opensuse and suse products. See bsc#1180699</a:documentation>
```
Comment (dgdavid) [inline control/control.rng:613]: NP: openSUSE and SUSE :stuck_out_tongue_winking_eye:

**Review by dgdavid — APPROVED** [review]

File: control/control.rnc [hunk 2]
```diff
@@ -294,6 +295,12 @@ precise_time =		        	element precise_time { BOOLEAN }
 ## Defines if makes sense for product to propose hibernation (jsc#SLE-12280)
 ## by default true
 propose_hibernation =	        	element propose_hibernation { BOOLEAN }
+## Defines list of default ntp servers from which product can pick.
+## Used mainly to distinguish between openSUSE and SUSE products. See bsc#1180699
+default_ntp_servers = element default_ntp_servers {
+  LIST,
+  element ntp_server { STRING }
```
Comment (teclator) [inline control/control.rnc:302]: Ouch, with this declaration it only permits one server.

Comment (teclator) [reply control/control.rnc:302]: And that is the reason for having this error https://github.com/yast/skelcd-control-openSUSE/pull/223/checks?check_run_id=3522806900 with the list below:

```xml
        <default_ntp_servers config:type="list">
          <ntp_server>0.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>1.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>2.opensuse.pool.ntp.org</ntp_server>
          <ntp_server>3.opensuse.pool.ntp.org</ntp_server>
        </default_ntp_servers>
```

Comment (dgdavid) [reply control/control.rnc:302]: > Ouch, with this declaration it only permits one server.

So, it should be 

```
element ntp_server { STRING }+
```

Shouldn't it?

Comment (teclator) [reply control/control.rnc:302]: `+` or `*` yep

**Pull request merged**